Commander/ArgumentDescription.swift

```diff
@@ -122,35 +122,52 @@ public class Options<T : ArgumentConvertible> : ArgumentDescriptor {
 
 public class Flag : ArgumentDescriptor {
   public typealias ValueType = Bool
-
+  
   public let name:String
+  public let falseName:String?
   public let flag:Character?
+  public let falseFlag:Character?
   public let description:String?
   public let `default`:ValueType
   public var type:ArgumentType { return .Option }
-
-  public init(_ name:String, flag:Character? = nil, description:String? = nil, `default`:Bool = false) {
+  
+  public init(_ name:String, falseName:String? = nil, flag:Character? = nil, falseFlag:Character? = nil, description:String? = nil, `default`:Bool = false) {
     self.name = name
+    self.falseName = falseName
```

Comment:
I think, we should here make the default value `no-\(name)` so when you explicitly set the disabled name there will be no `no-\(name)` handling.

```swift
self.disabledName = disabledName ?? "no-\(name)"
```